include/zephyr/xen/public/version.h

```diff
@@ -0,0 +1,117 @@
+/******************************************************************************
```

Comment:
Please add same SPDX identifier for line 1 as other public headers (MIT).

MAJOR: please take this header from stable XEN release (4.17.0), it looks different and already has this change.